code/game/jobs/jobs.dm

```diff
@@ -183,3 +183,19 @@ var/list/whitelisted_positions = list(
 
 	return titles
 
+var/global/list/exp_jobsmap = list(
+	EXP_TYPE_LIVING = list(), // all living mobs
+	EXP_TYPE_CREW = list(titles = command_positions + engineering_positions + medical_positions + science_positions + support_positions + supply_positions + security_positions + civilian_positions + whitelisted_positions), // crew positions

```

Comment:
Oh geez. Do we really not have a list for all positions? This seems like something that should be one of the globals in `jobs.dm`.